tests/fixtures/multiagent_decomposer/readme_test_case.hocon

```diff
@@ -27,7 +27,7 @@
     "use_direct": true,
 
     # Number of iterations that have to pass to call this test successful.
-    "success_ratio": "1/2",
+    "success_ratio": "1/1",
```

Comment:
Reduce the total number of iterations in the success_ratio for greater efficiency during tests.